contracts/token/SyndicateERC20.sol

```diff
@@ -295,7 +295,13 @@ contract SyndicateERC20 is AccessControl {
    */
   uint32 public constant ROLE_ERC20_SENDER = 0x0008_0000;
 
-  uint32 public constant ROLE_WHITE_LISTED_SPENDER = 0x0010_0000;
+  uint32 public constant ROLE_WHITE_LISTED_SPENDER = 0x0016_0000;
```

Comment:
this is hex, use 0x0020_0000 maybe be more fitting?

Reply:
I see but I do not understand why before they go 0002 0004 0008 and later 0010 0020. It does not make much sense for me. The problem is when you make the sum of it. I sense that there is an error there.